blackbox/stratisd.spec

```diff
@@ -56,12 +58,27 @@ mv %{buildroot}%{_bindir}/stratisd %{buildroot}%{_libexecdir}/stratisd
 %{__install} -Dpm0644 -t %{buildroot}%{_mandir}/man8 docs/stratisd.8
 %{__install} -Dpm0644 -t %{buildroot}%{_udevrulesdir} udev/61-stratisd.rules
 %{__install} -Dpm0644 -t %{buildroot}%{_unitdir} systemd/stratisd.service
+%{__install} -Dpm0644 -t %{buildroot}%{dracutdir}/dracut.conf.d dracut/90-stratis.conf
+mkdir -p %{buildroot}%{dracutdir}/modules.d/90stratis
+# dracut provides the 90stratis/module-setup.sh file
```

Comment:
Please remove the comment and add the install of ```module_setup.sh```. There should be no conflict (on f34).

Reply:
OK, I see it now.  Before, the `dracut` package had that file, and resulted in a conflict.  Now, `dracut` does not have that file, and `stratisd` must provide it.